website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -298,7 +298,7 @@ _Without a VCS repository_
       "vcs-repo-identifier": null,
       "working-directory": null,
       "workspace-kpis-runs-count": null,
-      "drift-detection": false
+      "assessments-enabled": false
```

Comment:
Again, I just think we need to mention that certain properties aren't available in TFE everywhere we're going to show them (refer to my comment above about this)